src/lib.ts

```diff
@@ -494,6 +498,36 @@ export function burnLibram(saveMp: number): void {
   }
 }
 
+export function bestFamiliarEquip(checkedFamiliar: Familiar): Item {
+  const validEquips = Item.all().filter(
+    (it) => have(it) && numericModifier(it, "Familiar Weight") > 0 && canEquip(checkedFamiliar, it),
+  );
+  if (validEquips.length === 0) return $item.none;
+
+  return validEquips.reduce((a, b) =>
+    numericModifier(a, "Familiar Weight") > numericModifier(b, "Familiar Weight") ? a : b,
+  );
+}
+
+export function chooseHeaviestEquippedFamiliar(checkedFamiliars?: Familiar[]): {
+  familiar: Familiar;
+  equip: Item;
+  expectedWeight: number;
+} {
+  return (checkedFamiliars ?? Familiar.all())
+    .filter(have)
+    .map((familiar) => {
+      const bestEquip = bestFamiliarEquip(familiar);
+
+      return {
+        familiar,
+        equip: bestEquip,
+        expectedWeight: familiarWeight(familiar) + numericModifier(bestEquip, "Familiar Weight"),
+      };
+    })
+    .reduce((a, b) => (a.expectedWeight > b.expectedWeight ? a : b));
+}
+
```

Comment:
This is nice. We can deprecate `chooseHeaviestFamiliar()` in `outfit.ts`.
Also, and I just noticed that it wasn't in `chooseHeaviestFamiliar()` either, we should add `!excludedFamiliars.includes(familiar.id)` to the filter if the user so wishes to not use said familiars

Reply:
Done